Form-Controls/styles.css

```diff
@@ -0,0 +1,99 @@
+@import url("https://fonts.googleapis.com/css2?family=Questrial&family=Roboto:wght@100&display=swap");
+
+body {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  min-height: 100vh;
+  font-family: "Questrial", sans-serif;
+  text-transform: uppercase;
+  background: rgb(224, 224, 231);
+  background: linear-gradient(
+    90deg,
+    rgba(224, 224, 231, 1) 0%,
+    rgba(158, 157, 157, 0.9276304271708683) 52%,
+    rgba(66, 70, 70, 1) 100%
+  );
+  /* font-family: "Roboto", sans-serif; */
+}
+main {
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  text-transform: uppercase;
+}
+#form {
+  margin-top: 150px;
+}
+#name,
+.name_label {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  width: 450px;
+  justify-content: space-evenly;
+  /* margin-bottom: 20px; */
+  gap: 20px;
+  row-gap: 10px;
+  margin-top: 10px;
+}
+#email_label {
+  padding-bottom: 10px;
+}
+#email_box > input {
+  width: 435px;
```

Comment:
Is a fixed width a good idea in layout? We never know what size our user's screen is, what text or zoom level they are using, and, very often, what else is on the page. 

Consider a fluid and relational strategy where you rarely size anything, just set rules about maximum sizes, and usually on the highest parent container you can. Try setting all inputs to fill their container: width:100%. Then, size the container if you need to.